Sift/Response/ScoreResponse.cs

```diff
@@ -14,6 +14,9 @@ public class ScoreResponse : SiftResponse
         [JsonProperty("latest_labels")]
         public Dictionary<string, LabelJson> LatestLabels { get; set; }
 
+        [JsonProperty("workflow_statuses")]
+        public List<WorkflowStatusResponse> workflowStatuses { get; set; }
```

Comment:
the property name should be pascal-case `WorkflowStatuses`

Reply:
Doh!